lib/internal/bootstrap_node.js

```diff
@@ -101,10 +101,14 @@
         delete process.env.NODE_UNIQUE_ID;
       }
 
+      var internalModule = NativeModule.require('internal/module');
```

Comment:
Can you move this into the branches of the `if` statement below?  It avoids the hit of loading the module unless it's actually used.